### PR TITLE
Upgrade packages

### PR DESCRIPTION
**Pull request by Kampfheizung**
- @mdx-deck/themes to 4.1.0
- mdx-deck to 4.1.1
- react-syntax-highlighter to 13.5.1

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: node:8
+      - image: node:12.8
```
Comment (Tobi-mmt) [inline .circleci/config.yml:6]: why not 12.18 ?

Comment (Tobi-mmt) [reply .circleci/config.yml:6]: or even better: `node:lts`
https://circleci.com/developer/images/image/cimg/node
Bildschirmfoto 2020-09-03 um 11 13 45

Comment (Kampfheizung) [reply .circleci/config.yml:6]: oh that's a typo :D i wanted to type 12.18 but missed the `1`. I will use lts 👍

Comment (Kampfheizung) [reply .circleci/config.yml]: done 👍

**Review by Tobi-mmt — APPROVED** [review]

**Pull request merged**